Combobox/ComboPopup.js

```diff
@@ -61,13 +67,39 @@ define([
 				}
 			}
 		},
+		
+		/**
+		 * Event handler called when a event of type "keynav-child-navigated" is dispatched
+		 * on the combobox.list element.
+		 * @param Event evt The Event instance. 
+		 */
+		_onKeynavChildNavigatedHandler: function (evt) {
+			var navigatedChild = evt.newValue;
+			this._setActiveDescendant(navigatedChild);
+		},
+		
+		/**
+		 * Sets the active descendant on the inputNode.
+		 * @param String nd The select node.
+		 * @protected
+		 */
+		_setActiveDescendant: function (nd) {
+			if (nd) {
+				if (!nd.id) {
+					nd.id = "d-combobox-item-" + idCounter++;
+				}
+
+				this.inputNode.setAttribute("aria-activedescendant", nd.id);
+			}
+		},
```

Comment:
This code seems mainly useless since the template already has this on the `<input>`:

```
aria-activedescendant="{{combobox.list.navigatedDescendant}}"
```

The one new piece of functionality is line 89, about setting ids on the list nodes, but that code doesn't work because there's no `idCounter` variable.

Reply:
Perhaps the code in the template isn't working because it doesn't realize when combobox.list.navigatedDescendant changes?